src/Encamina.Enmarcha.SemanticKernel/Internals/CosineStringSimilarityComparer.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Numerics.Tensors;
+
+using Encamina.Enmarcha.AI.Abstractions;
+
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace Encamina.Enmarcha.SemanticKernel.Internals;
+
+#pragma warning disable SKEXP0001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Any `pragmas` that disables warnings goes at the beginning and end of the code. Remember to separate the `pragma`s block with a blank line before the code block.

The only exception is when they apply to a few lines of code within the file